Source/Diagnostics/BoostedFrameDiagnostic.cpp

```diff
@@ -446,6 +444,36 @@ namespace
 }
 #endif
 
+namespace
+{
+    void
+    CopySlice(MultiFab& tmp, MultiFab& buf, int k_lab, 
+              std::vector<int> map_actual_fields_to_dump)
```

Comment:
It's probably better to pass this vector by const reference.

Reply:
I changed `std::vector<int> map_actual_fields_to_dump` to `const std::vector<int>& map_actual_fields_to_dump` in the last commit. Is this what you meant?